app/Http/Controllers/MonitorController.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+require_once 'include/common.php';
+
+use DateTime;
+use DateInterval;
+use DatePeriod;
+use App\Enums\ClassicPalette;
+use App\Enums\HighContrastPalette;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\DB;
+use Symfony\Component\HttpFoundation\Response;
+use Illuminate\Http\Request;
+
+class MonitorController extends AbstractController
+{
+    /**
+     * Get statistics about our recent submissions queue performance.
+     */
+    public function get(): JsonResponse
+    {
+        $user = Auth::user();
+        if ($user === null || !$user->IsAdmin()) {
```

Comment:
`IsAdmin()` checks for the case where the user is not logged in.

```suggestion
        if (!$user->IsAdmin()) {
```

Reply:
Auth::user() returns null when the user isn't logged in, so we need this check to avoid `Call to a member function IsAdmin() on null`

https://laravel.com/api/9.x/Illuminate/Support/Facades/Auth.html#method_user

Reply:
Oops, you're right.  I don't know what I was thinking when I wrote that...

Removing the null check is fine for endpoints wrapped in the `auth` middleware, but none of the API endpoints use auth middleware yet so it is still necessary.